components/services/markdown/src/index.js

```diff
@@ -28,6 +28,14 @@ class ServiceMarkdown extends Component {
     )
   }
 
+  componentDidUpdate() {
```

Comment:
componentDidUpdate will be called every time the parent component is rendered, even if your props are unchanged. So, you will rescroll over and over on each update. You must control that.
The scroll should be done only html changes.
So you can fix this with a shouldComponentUpdate.